ca_tools/launch/keyboard_teleop.launch

```diff
@@ -1,3 +1,5 @@
+<!-- USE MOVE_ROBOT VARIABLE TO SELECT WHICH ROBOT TO CONTROL, DEFAULT=1 -->
 <launch>
-  <node pkg="ca_tools" type="key_teleop.py" name="keyboard_teleop" output="screen"/>
+  <arg name="ID" default="$(optenv ID 1)"/>
+   <node pkg="ca_tools" type="key_teleop.py" name="keyboard_teleop" output="screen" ns="$(eval 'create' + str(arg('ID')))" if="$(eval (arg('ID') > 0) and str(arg('ID')).isdigit())" />
```

Comment:
Invert the check:
```
"$(eval str(arg('ID')).isdigit() and (arg('ID') > 0) )"
```
Why? Because if you send something like this `....launch ID:="1"`, it will fail checking `>0` because it's not a string.

Reply:
Adressed